benchmark/make_epiaware_suite.jl

```diff
@@ -3,7 +3,28 @@
 """
 function make_epiaware_suite(model; check = true,
         adbackends = [:forwarddiff, :reversediff, :reversediff_compiled])
-    suite = TuringBenchmarking.make_turing_suite(
-        model; check = check, adbackends = adbackends)
+    suite = prefix_warnings(
+        () -> TuringBenchmarking.make_turing_suite(
+            model; check = check, adbackends = adbackends),
+        model
+    )
     return suite
 end
+
+function prefix_warnings(f, g)
```

Comment:
Can you provide more descriptive variable names?

Reply:
I mean yes in principle we can. I wonder if we need to for an internal function that is stored directly below its use case? That could just be letting it being late here talk though...

Reply:
Better names and type annotations would help me understand what the code is doing.

Reply:
I'm think this is fine for an internal function but if you feel strongly we can reopen.